Postgres-9.5.Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM postgres:9.5
 ARG POSTGRES_VERSION=9.5
 
 RUN apt-get update --fix-missing && \
-    apt-get install -y postgresql-server-dev-$POSTGRES_VERSION postgresql-$POSTGRES_VERSION-repmgr wget  openssh-server
+    apt-get install -y postgresql-server-dev-$POSTGRES_VERSION postgresql-$POSTGRES_VERSION-repmgr wget  openssh-server barman-cli
```

Comment:
`barman-cli` consists of some utilities for remote execution of `recover_command` and also `rsync`, which is a must have for recovery. In documentation, I propose to copy all WALs with base backup in one step, but remote copy on demand is also an option.